akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -179,7 +175,21 @@ object ShardRegion {
   /**
    * Reply to `GetCurrentRegions`
    */
-  @SerialVersionUID(1L) final case class CurrentRegions(regions: Set[Address]) {
+  @SerialVersionUID(2L) final case class CurrentRegions(regionActorPaths: Set[ActorPath]) {
```

Comment:
this is unfortunately not binary backwards compatible (and I don't think we should squeeze it into 2.4.0)